tests/setup/sg_init.sh

```diff
@@ -73,6 +73,7 @@ if [ "${IS_ES_SECURED:-}" = true ] ; then
     cp ${TMP_DIR}/search-guard-ssl/example-pki-scripts/transport/node-0-keystore.jks ${ES_HOME}/plugins/search-guard-2/sgconfig/transport-node-0-keystore.jks
     cd ${ACTUAL_DIR}
 
+    cp ${ES_CONF}/sg_*.yml ${ES_HOME}/plugins/search-guard-2/sgconfig
```

Comment:
Aren't SG config files installed automatically when the SG plugin is [installed](https://github.com/ViaQ/watches-cli/blob/2f58f73a4234ee3578bf490e79edce4231fb850c/tests/setup/sg_init.sh#L65) using `${ES_HOME}/bin/plugin install -b com.floragunn/search-guard-2/${SG_VER}` command?

Or these SG config files are different from those provided with SG by default?